scaffold-api/src/scaffold_api/resources/apihelper.py

```diff
@@ -26,10 +31,70 @@ class Api(BaseApi):
     @property
     def specs_url(self):
         """Return URL for endpoint."""
-        scheme = 'http' if '5000' in self.base_url else 'https'
-        return url_for(self.endpoint('specs'), _external=True, _scheme=scheme)
+        scheme = "http" if "3200" in self.base_url else "https"
+        return url_for(self.endpoint("specs"), _external=True, _scheme=scheme)
+
+    @classmethod
+    def common_decorator(cls, api, endpoint_description):
+        """Common decorators for the resources"""
+
+        def decorator(func):
+            @wraps(func)
+            @cross_origin(origins=allowedorigins())
+            @auth.require
```

Comment:
maybe we have the auth things in another common decorator? they are optional and I remember in met we had things like auth.require auth.optional and auth.has_one_of_roles. Our auth needs might become more complex and might not fit in a common decorator